app/components/personal_details_review_component.rb

```diff
@@ -0,0 +1,20 @@
+class PersonalDetailsReviewComponent < ActionView::Component::Base
+  validates :application_form, presence: true
+
+  def initialize(application_form:)
+    @application_form = application_form
+    @personal_details_form = CandidateInterface::PersonalDetailsForm.build_from_application(
```

Comment:
Following on from our conversations about reusing this component in the provider interface, we shouldn't do that until `CandidateInterface::PersonalDetailsReviewPresenter` has been moved into the top-level of the application. Otherwise we're crossing the boundaries between namespaces.

Reply:
Agree 👍